<launch>
   <node pkg="homography_servo" name="homography_servo" type="homography_servo_node" output="screen">
      <rosparam command="load" file="$(find homography_servo)/config/config.yaml"/>
      <!-- <remap from="/mavros/setpoint_raw/attitude" to="/test"/> -->
   </node>
</launch>